.env.example

```diff
@@ -32,3 +32,6 @@ DB_NAME=""
 DB_USER=""
 DB_PASSWORD=""
 DB_HOST=""
+
+REDIS_HOST=""
+REDIS_PORT=""
```

Comment:
Rather than put this at the bottom, let's keep the file in the order of the settings files themselves. So, this should go between pacer.py and sentry.py, and we should add a comment like this:

```
# redis.py
REDIS_HOST=""
REDIS_PORT=""
```

Reply:
Got it!